client/src/styles/Multiselect.scss

```diff
@@ -1,128 +1,169 @@
-  position: absolute;
-  right: 10px;
-  top: 50%;
-  transform: translateY(-50%);
-  width: 14px;
-}
-.icon_down_dir:before {
-  content: "\e803";
-}
-.custom-close {
-  display: flex;
-}
 
-// JF custom styles
+  &.disable_ms,
+  .disable_ms {
+    pointer-events: none;
+    opacity: 0.5;
+  }
+  &.display-none,
+  .display-none {
+    display: none;
+  }
+  .searchWrapper {
+    border: 1px solid #cccccc;
+    border-radius: 4px;
+    padding: 5px;
+    min-height: 22px;
+    position: relative;
+    &.hasError {
+      outline: 2px $justfix-orange solid;
+      outline-offset: -2px;
+    }
+  }
+  input {
+    border: none;
+    margin-top: 3px;
+    background: transparent;
+    &:focus {
+      outline: none;
+    }
+  }
+  .chip {
+    padding: 4px 10px;
+    background: #0096fb;
+    margin-right: 5px;
+    margin-bottom: 5px;
+    border-radius: 11px;
+    display: inline-flex;
+    align-items: center;
+    font-size: 1.4rem;
+    line-height: 100%;
+    color: #fff;
+    white-space: nowrap;
+  }
+  .singleChip {
+    background: none;
+    border-radius: none;
+    color: inherit;
+    white-space: nowrap;
+    i {
+      display: none;
+    }
+  }
+  .optionListContainer {
+    position: absolute;
+    width: 100%;
+    background: #fff;
+    border-radius: 4px;
+    z-index: 2;
+    ul {
+      display: block;
+      padding: 0;
+      margin: 0;
+      border: 1px solid #ccc;
+      border-radius: 4px;
+      max-height: 250px;
+      overflow-y: auto;
+      li {
+        display: flex;
+        font-size: 1.6rem;
+        gap: 1.6rem;
+        padding: 0.6rem 1.2rem;
+        &:hover {
+          cursor: pointer;
+          text-decoration: underline;
+        }
+        &.groupHeading {
+          color: #908e8e;
+          pointer-events: none;
+          padding: 5px 15px;
+        }
+        &.groupChildEle {
+          padding-left: 30px;
+        }
+        input {
+          --box-size: 1.8rem;
+
+          -webkit-appearance: none;
+          -moz-appearance: none;
+          -o-appearance: none;
+          appearance: none;
+
+          position: relative;
+          border: 0.1rem solid $justfix-black;
+          min-height: calc(var(--box-size));
+          min-width: calc(var(--box-size));
+          height: calc(var(--box-size));
+          width: calc(var(--box-size));
+          border-radius: 0.4rem;
+          margin: 0;
+
+          &:checked {
+            background-color: $justfix-black;
+            color: $justfix-white;
+            border-color: $justfix-black;
+            &:checked:before {
+              content: "";
+              position: absolute;
+              left: 0.6rem;
+              top: 0.2rem;
+              width: 0.5rem;
+              height: 1rem;
+              border: solid $justfix-white;
+              border-width: 0 0.2rem 0.2rem 0;
+              transform: rotate(45deg);
+            }
+          }
+          span {
+            align-self: center;
+          }
```

Comment:
This is the only section that was actually changed, everything else was just nested under the `multiselectContainer` class.